conf/application.conf

```diff
@@ -83,6 +83,14 @@ microservice {
   hosts.allowList.1 = "www.tax.service.gov.uk"
 
   services {
+    access-control {
+      enabled = "true"
+      allow-list = [
+        "coronavirus-job-retention-scheme-frontend",
+        "coronavirus-jrs-helpdesk-frontend"
+      ]
+    }
+
```

Comment:
This should be a default in our application.config, we should only have this in QA/Prod

Reply:
Do you mean the whole allow-list block should be in app-config-base?

Reply:
We could put it there, I don't think it should be app config but maybe I'm wrong?